postgres-appliance/build_scripts/base.sh

```diff
@@ -146,6 +146,22 @@ for version in $DEB_PG_SUPPORTED_VERSIONS; do
         done
     )
 
+    if [ "${TIMESCALEDB_APACHE_ONLY}" != "true" ] && [ "${TIMESCALEDB_TOOLKIT}" = "true" ]; then
+        __versionCodename=$(sed </etc/os-release -ne 's/^VERSION_CODENAME=//p')
```

Comment:
maybe just `$(lsb_release -c -s)` ?

Reply:
I didn't use that because lsb_release isn't included in standard Ubuntu docker images.
Doesn't look like anything in the spilo image does either, so probably better just getting it from the text file.
Agree?

```
++ lsb_release -c -s
base.sh: line 150: lsb_release: command not found
```

Reply:
sorry, you're right. It is installed later during Patroni installation